Formula/s/subversion.rb

```diff
@@ -199,7 +199,11 @@ def install
     unless perl_extern_h.exist?
       # No EXTERN.h, maybe it's system perl
       perl_version = Utils.safe_popen_read(perl.to_s, "--version")[/v(\d+\.\d+)(?:\.\d+)?/, 1]
-      perl_core = MacOS.sdk_path/"System/Library/Perl"/perl_version/"darwin-thread-multi-2level/CORE"
+      perl_core = if OS.mac?
+        MacOS.sdk_path/"System/Library/Perl"/perl_version/"darwin-thread-multi-2level/CORE"
+      else
+        "System/Library/Perl"/perl_version/"darwin-thread-multi-2level/CORE"
+      end
       perl_extern_h = perl_core/"EXTERN.h"
     end
```

Comment:
`else` block doesn't make sense on Linux. I'd just make this whole block `if OS.mac? && !perl_extern_h.exist?`

(see next two suggestions)